### PR TITLE
DM-1357 Add LWM2M 1.1 information to What's New

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
Adding LWM2M 1.1 to What's new

### CODE REVIEW

**Comment by BeateRixen** [comment]
@sag-eweingaertner 
Both entries are already part of the device management/improvements section of the release notes: https://staging-resources.cumulocity.com/releasenotes/release-10-15-0/devices-10-15-0/
image

Usually we use the Whats New section for larger epics which we want to promote more prominently. In these cases, we do not have a duplicate entry in the improvements. I recommend to either place it here if it is a major achievement and remove the entries from the improvements or leave it in the improvements only.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
...
+### LWM2M 1.1 support
+
+We are now supporting the following LWM2M 1.1 features:
+
+- LWM2M Send
+- Composite Operations (read, write, observe, cancel observation)
+- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
+- New LWM2M data formats: Core Link, Unsigned Integers
+
+For more information on our 1.1 support, see our [LwM2M protocol integration guide](http://localhost:58835/guides/protocol-integration/lwm2m/).
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:46]: The links point to your local environment.

**Comment by sourajitsaha17** [comment]
.DS_store was committed

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 1 above)
Comment (sag-eweingaertner) [reply content/release-10-15-0/whatsnew-10-15-0.md:46]: Whoopsie!

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 2]
```diff
@@ -59,3 +51,26 @@ The video snippets below demonstrate these new wizards.
...
+We are now supporting the following LWM2M 1.1 features:
+
+- LWM2M Send
+- Composite Operations (read, write, observe, cancel observation)
+- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
+- New LWM2M data formats: Core Link, Unsigned Integers
+
+For more information on our 1.1 support, see our [LwM2M protocol integration guide](/guides/protocol-integration/lwm2m/).
+
+#### LWM2M time timestamp conversion
```
Comment (Oezge-Akin) [inline content/release-10-15-0/whatsnew-10-15-0.md:66]: LWM2M device object timestamp conversion?

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 3]
```diff
@@ -59,3 +51,26 @@ The video snippets below demonstrate these new wizards.
...
+
+If a timestamp (resources 5518 and 6050, resource 5 for object 6) is received together with resources of the same object, this timestamp can be used as creation timestamp for measurements, events and alarms. See [Advanced settings](/guides/protocol-integration/lwm2m/#advanced-settings) for details.
```
Comment (Oezge-Akin) [inline content/release-10-15-0/whatsnew-10-15-0.md:68]: Maybe you can add it needs to be enabled.
See [Advanced settings](...) for details and to enable during device registration.

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 2 above)
Comment (sag-eweingaertner) [reply content/release-10-15-0/whatsnew-10-15-0.md]: fixed

**Comment by sag-eweingaertner** [comment]
> 

You are right. I will remove it then from What's new.

**Comment by sag-eweingaertner** [comment]
> .DS_store was committed

Fixed.

**Review by Oezge-Akin — APPROVED** [review]

**Comment by sag-eweingaertner** [comment]
> @sag-eweingaertner Both entries are already part of the device management/improvements section of the release notes: https://staging-resources.cumulocity.com/releasenotes/release-10-15-0/devices-10-15-0/ image
> 
> Usually we use the Whats New section for larger epics which we want to promote more prominently. In these cases, we do not have a duplicate entry in the improvements. I recommend to either place it here if it is a major achievement and remove the entries from the improvements or leave it in the improvements only.

Removed with latest commit

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 4]
```diff
@@ -34,18 +34,10 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
...
-
-For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
-
-
+### Device Management
 ### Revised LWM2M device registration
 
-The LWM2M agent now provides revised single and bulk device registration wizards. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.
+The LWM2M agent now provides now registration wizards for both single and bulk LWM2M device registration. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:40]:
```suggestion
The LWM2M agent now provides registration wizards for both single and bulk LWM2M device registration. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/10.15.0/protocol-integration/lwm2m/#register-device) in the *Protocol integration guide*.
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 5]
```diff
@@ -58,3 +50,22 @@ The video snippets below demonstrate these new wizards.
 <video width="99%" controls>
   <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
 </video>
+
+#### LWM2M 1.1 support
+
+We are now supporting the following LWM2M 1.1 features:
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:56]:
```suggestion
We now support the following LWM2M 1.1 features:
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 6]
```diff
@@ -58,3 +50,22 @@ The video snippets below demonstrate these new wizards.
 <video width="99%" controls>
   <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
 </video>
+
+#### LWM2M 1.1 support
```
Comment (Oezge-Akin) [inline content/release-10-15-0/whatsnew-10-15-0.md:54]: `####` -> `###`

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 7]
```diff
@@ -58,3 +50,22 @@ The video snippets below demonstrate these new wizards.
...
+
+We are now supporting the following LWM2M 1.1 features:
+
+- LWM2M Send
+- Composite Operations (read, write, observe, cancel observation)
+- New LWM2M serialization formats: CBOR, SenML JSON, SenML CBOR
+- New LWM2M data formats: Core Link, Unsigned Integers
+
+For more information on our 1.1 support, see our [LwM2M protocol integration guide](https://cumulocity.com/guides/protocol-integration/lwm2m/).
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:63]:
```suggestion
For more information on our 1.1 support, refer to [LWM2M](https://cumulocity.com/guides/10.15.0/protocol-integration/lwm2m/) in the *Protocol integration guide*.
```

Comment (BeateRixen) [reply content/release-10-15-0/whatsnew-10-15-0.md:63]: Perhaps add the section here?

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 6 above)
Comment (sag-eweingaertner) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Fixed

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 8]
```diff
@@ -34,18 +34,10 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
...
-
-The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
-
-![All devices](/images/release-notes/multiple_lns_connection.png)
-
-For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
-
-
+### Device Management
 ### Revised LWM2M device registration
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:38]: This and the following headings should have heading level 4 ####

Comment (sag-eweingaertner) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Done

**Review by sourajitsaha17 — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**